main.nf

```diff
@@ -15,7 +15,10 @@ nextflow.enable.dsl = 2
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-params.fasta = WorkflowMain.getGenomeAttribute(params, 'fasta')
+params.fasta        = WorkflowMain.getGenomeAttribute(params, 'fasta')
+params.fasta_fai    = WorkflowMain.getGenomeAttribute(params, 'fai')
+params.dict         = WorkflowMain.getGenomeAttribute(params, 'dict')
+params.bwa          = WorkflowMain.getGenomeAttribute(params, 'bwa')
```

Comment:
You need an aligner in a variant calling wf?

Reply:
Yes `gridss` uses Whole Genome Assembly to find structural variants which needs remapping with `bwa`